### PR TITLE
Enable IPU path 8.10 -> 9.5

### PR DESCRIPTION
**Pull request by matejmatuska**
* Enable 9.5 target
* Add RHEL 9.5 product certificates

### CODE REVIEW

File: repos/system_upgrade/common/files/upgrade_paths.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+        "8": ["9.2", "9.4", "9.5"]
     },
     "saphana": {
         "7.9": ["8.10", "8.8"],
         "7": ["8.10", "8.8"],
         "8.8": ["9.2"],
-        "8.10": ["9.4"],
-        "8": ["9.4", "9.2"]
+        "8.10": ["9.4", "9.5"],
+        "8": ["9.4", "9.2", "9.5"]
```
Comment (pirat89) [inline repos/system_upgrade/common/files/upgrade_paths.json:14]: @matejmatuska  sap hana is not supported for odd. it sould stay untouched.

**Review by pirat89 — CHANGES_REQUESTED** [review]

**Review by pirat89 — APPROVED** [review]
lgtm

**Comment by pirat89** [comment]
@matejmatuska  seems we cannot allow it yet. so let's drop the change in tests and put it into a separate PR.

**Comment by matejmatuska** [comment]
> @matejmatuska seems we cannot allow it yet. so let's drop the change in tests and put it into a separate PR.

Ok, I dropped the commit.

**Review by pirat89 — APPROVED** [review]
lgtm

**Pull request merged**